src/main/java/com/jagrosh/jmusicbot/commands/owner/PlaylistCmd.java

```diff
@@ -169,6 +176,67 @@ protected void execute(CommandEvent event)
             }
         }
     }
+
+    public class AppendlistqCmd extends OwnerCommand 
+    {
+        public AppendlistqCmd()
+        {
+            this.name = "appendqueue";
+            this.aliases = new String[]{"addqueue"};
+            this.help = "appends songs in queue to an existing playlist";
+            this.arguments = "<name>";
+            this.guildOnly = false;
+        }
+
+        @Override
+        protected void execute(CommandEvent event) 
+        {
+            String pname = event.getArgs().replaceAll("\\s+", "_");
+            Playlist playlist = bot.getPlaylistLoader().getPlaylist(pname);
+            if(playlist==null)
+                event.replyError(event.getClient().getError()+" Playlist `"+pname+"` doesn't exist!");
+            else
+            {
+                AudioHandler ah = (AudioHandler)event.getGuild().getAudioManager().getSendingHandler();
+                List<QueuedTrack> list = ah.getQueue().getList();
+                if(list.isEmpty())
+                {
+                    Message nowp = ah.getNowPlaying(event.getJDA());
+                    if(nowp==null)
+                    {
+                        event.replyWarning(event.getClient().getWarning() + " There is no music in the queue!");
+                        return;
+                    }
+                }
+                List<AudioTrack> al = new ArrayList<>();
+                al.add(ah.getPlayer().getPlayingTrack());
+                for(QueuedTrack q: list)
+                {
+                    al.add(q.getTrack());
+                }
+                StringBuilder builder = new StringBuilder();
+                playlist.getItems().forEach(item -> builder.append("\r\n").append(item));
+                for(AudioTrack url: al)
+                {
+                    if(url==null)
+                    {
+                        continue;
+                    }
+                    String u = url.getInfo().uri;
+                    builder.append("\r\n").append(u);
+                }
+                try
+                {
+                    bot.getPlaylistLoader().writePlaylist(pname, builder.toString());
+                    event.replySuccess(event.getClient().getSuccess()+" Successfully added "+al.size()+" items to playlist `"+pname+"`!");
+                }
+                catch(IOException e)
+                {
+                    event.replyError(event.getClient().getError()+" I was unable to append to the playlist: "+e.getLocalizedMessage());
```

Comment:
... and finally here
```suggestion
                    event.replyError("I was unable to append to the playlist: "+e.getLocalizedMessage());
```